ui/apps/platform/cypress/helpers/networkGraphPF.js

```diff
@@ -0,0 +1,111 @@
+import * as api from '../constants/apiEndpoints';
+import { selectors as networkGraphSelectors } from '../constants/NetworkPage';
```

Comment:
Recommendation not a blocker to merge:

If there is a helper function for a semantic interaction, prefer to encapsulate the relevant api endpoints and DOM selectors in the helpers file.
* Only Network Graph tests`api.network.whatever` properties. Most if not all deserve to be encapsulated by visit or interact helper functions.
* For selectors, the only counter-example that comes to mind for **both** helpers **and** tests to use selectors is disabled/enabled assertions about Save or Test buttons in integration form tests.